src/02-data-types.js

```diff
@@ -22,7 +24,9 @@ function getLengthOfString(str) {}
  *  convertToNumber("111");
  *  //> 111
  */
-function convertToNumber(val) {}
+function convertToNumber(val) {
+  return Number(parseInt(val));
```

Comment:
Be careful with calling `parseInt()` with numbers.  What would happen if we passed in `12.5` to our function?  Simply calling a `Number` constructor is enough to insure we have a number returned from the function.